UnitTests/GitUI.Tests/UserControls/FilterInfoTests.cs

```diff
@@ -1183,10 +1183,10 @@ string GetFilterRefName(string gitRef)
 
         [TestCase("message1", true)]
         [TestCase("message1 --not", true)]
-        [TestCase("--not=message1", true)]
+        [TestCase("--not=message1", false)]
         [TestCase("--not message1", false)]
         [TestCase(" --not message1", true)]
```

Comment:
Is this intended? Else `.TrimLeft().StartsWith("--")`?

Reply:
(unchanged from before)
The message to search for is not trimmed, initial and trailing whitespace is searched for.